File-Formats/Presentation/Presentation-to-image.md

```diff
@@ -17,6 +17,9 @@ This section covers converting an entire Presentation or a single slide to image
 * JPEG
 * PNG
 
+To quickly start converting a PowerPoint Presentation to Images, please check out this video:
```

Comment:
To quickly start converting a PowerPoint Presentation to images, please check out this video:

Reply:
modified